resources/js/processes/designer/Assets.vue

```diff
@@ -8,90 +8,237 @@
     <div class="mt-3">
       <b-card-group deck>
         <b-card
+          v-for="(asset, index) in assets.slice(0, 3)"
+          :key="index"
           bg-variant="light"
           class="text-center"
+          @click="toggleButtons(index)"
         >
-          <asset
-            color="#4DA2EB"
-            icon="fas fa-play-circle"
-            asset_name="Processes"
-          />
-        </b-card>
-        <b-card
-          bg-variant="light"
-          class="text-center"
-        >
-          <asset
-            color="#8EB86F"
-            icon="fas fa-file-alt"
-            asset_name="Screens"
-          />
-        </b-card>
-        <b-card
-          bg-variant="light"
-          class="text-center"
-        >
-          <asset
-            color="#F7CF5D"
-            icon="fas fa-code"
-            asset_name="Scripts"
-          />
+          <div v-show="!showButtons[index]">
+            <asset
+              :color="asset.color"
+              :icon="asset.icon"
+              :asset_name="asset.asset_name"
+            />
+          </div>
+
+          <div
+            v-show="showButtons[index]"
+            class="text-left"
+          >
+            <asset-buttons
+              :asset_name_all="asset.asset_name_all"
+              :variant="variant"
+              :urlPath="asset.urlPath"
+            >
+              <template v-slot:select-template-slot>
+                <div v-if="asset.flag === 1">
```

Comment:
hardcode 1

Reply:
TODO: The section below must be refactored in next tickets to avoid hard code in v-for